src/app/clarity/descriptions/ClarityDescriptions.tsx

```diff
@@ -0,0 +1,82 @@
+import { RootState } from 'app/store/types';
+import { useSelector } from 'react-redux';
+import { Line, LinesContent } from '../descriptionInterface';
+/* eslint-disable css-modules/no-unused-class */
+import styles from './Description.m.scss';
+
+const customContent = (content: LinesContent) => {
+  if (content.linkUrl) {
+    return (
+      <a href={content.linkUrl} target="_blank">
```

Comment:
We have an `ExternalLink` component you can use for this.

Reply:
I will take a look